picker-starter/src/components/Picker/Picker.vue

```diff
@@ -90,6 +99,10 @@ export default {
       )
     },
   },
+  created() {
+    const { errorNotification } = useErrorNotification()
+    this.errorNotification = errorNotification
+  },
```

Comment:
Just a nit as I told you yesterday about `setup()`:
```suggestion
  setup() {
    const { errorNotification } = useErrorNotification()
    return { errorNotification }
  },
```